jobs-in-tech.md

```diff
@@ -0,0 +1,18 @@
+## Jobs in Tech
+Below are some descriptions of the various types of roles that you might see when looking for a job. These descriptions are based on the experiences of alumni who have had them, and focus on what sorts of things you might expect from a junior level position. As with any industry, titles can have quite a fluid meaning and each company will attribute a slightly different set of responsibilities each title.
+
+It's worth making sure that you're applying to a role that sets you in the right direction given your career goals, or that you at least have a good understanding of what that type of job is likely to entail.
+
+## Software developer
+
+## Tester
```

Comment:
Just googled the difference between QA and tester, quite cool! Wonder if we should keep the trend and have `Test engineer` instead of tester?

Reply:
Good point, I'll change that :)